.env.example

```
@@ -112,3 +102,11 @@ NEXT_PUBLIC_BASE_URL="http://localhost:3000"
 
 # Node environment
 NODE_ENV="development"
+
+# ==============================================================================
+# E2EE - Key Wrapping
+# ==============================================================================
+
+# 32-byte hex secret for server-side private key wrapping
+# Generate with: openssl rand -hex 32
+KEY_WRAPPING_SECRET=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)